lapis2/README.md

```diff
@@ -2,6 +2,13 @@
 
 This is a generalized Lightweight API for Sequences. It uses [SILO](https://github.com/GenSpectrum/LAPIS-SILO) as database for storing and querying the sequence data.
 
+## OpenAPI documentation
+
+The swagger ui is available at `url.to.lapis:<port>/swagger-ui.html`.
+
+The OpenAPI specification is available at `url.to.lapis:<port>/v3/api-docs` (in JSON format) or at 
+`url.to.lapis:<port>/v3/api-docs.yaml` (in YAML format).
```

Comment:
Why `v3`?

Reply:
Is there an easy way to change it? Our own API endpoints (in the current LAPIS) are prefixed with `/v1` and it's a bit misleading to have `v3` there.

Reply:
Ok true, I configured it to `/api-docs`